packages/data-stores/src/help-center/actions.ts

```diff
@@ -130,6 +129,16 @@ export const setShowMessagingChat = function* () {
 	yield resetStore();
 };
 
+export const setShowSupportDoc = function* ( link: string, postId: number ) {
+	const params = new URLSearchParams( {
+		link,
+		postId: String( postId ),
+		cacheBuster: String( Date.now() ),
```

Comment:
This unused field is there, as the name implies, to "bust" "cache" - without it, if the user clicks on a support link, Help Center opens, etc. But if they navigate (back, to a different doc, etc.) within Help Center, clicking that support link in the main interface will not do anything (because the `initialRoute` value will be the same - no state change, no `useEffect` trigger). This will ensure it's always triggered without the need to revamp how we store the value or using more complicated approaches. But open to suggestions if there's a better option I missed 🙇‍♂️